java-ast/tests/src/test/java/org/jboss/windup/ast/java/test/JavaASTReferenceResolverTest.java

```diff
@@ -32,7 +32,7 @@ public void testHelloWorld()
                                 5, 23, 4,
                                 "public static void main(String[] argv) throws Exception {")));
```

Comment:
I am  missing here negative checks like 
```Assert.assertFalse(references.contains(new ClassReference("testclasses.helloworld.HelloWorld.main(String[])", "testclasses.MYEXTRAPACKAGE.helloworld", "HelloWorld", "main",
                                ResolutionStatus.RESOLVED, TypeReferenceLocation.METHOD,
                                5, 23, 4,
                                "public static void main(String[] argv) throws Exception {")));```
To verify that it really returns a false on different package , and the same with the class "HelloWorldX".